source/particle/property/interface.cc

```diff
@@ -393,10 +393,21 @@ namespace aspect
                   else
                     found_cell = cell;
 
-                  const std::vector<std::vector<double> > interpolated_properties = interpolator.properties_at_points(particle_handler,
-                                                                                    std::vector<Point<dim> > (1,particle_location),
-                                                                                    ComponentMask(property_information.n_components(),true),
-                                                                                    found_cell);
+                  std::vector<std::vector<double> > interpolated_properties;
+
+                  try
+                    {
+                      interpolated_properties = interpolator.properties_at_points(particle_handler,
+                                                                                  std::vector<Point<dim> > (1,particle_location),
+                                                                                  ComponentMask(property_information.n_components(),true),
+                                                                                  found_cell);
+                    }
+                  catch (const std::exception &e)
+                    {
+                      std::cerr << e.what() << '\n';
```

Comment:
I think it makes sense to use std::endl instead, to make sure output is flushed.